website/docs/commands/taint.html.markdown

```diff
@@ -41,7 +41,8 @@ the output from other commands, such as:
 
  * `aws_instance.foo`
  * `aws_instance.bar[1]`
- * `module.foo.module.bar.aws_instance.baz`
+ * `aws_instance.baz[\"key\"]`
```

Comment:
Are those escapes always required, or is that a shell-specific escaping? I'm curious if a user could also run `terraform taint 'aws_instance.baz["key"]'`.

Reply:
yes, I think single quotes are the more universal mechanism here (but I cannot recall what windows cmd does). Unfortunately it's always somewhat shell specific.

Reply:
Yeah I prefer the consistency with the other examples (not single quoting), and a hint will be helpful

Reply:
Based on my understanding of standard Windows command line processing (from implementing [`shquot`](https://godoc.org/github.com/apparentlymart/go-shquot/shquot)), I think this should work both there and in POSIX shells. It might _not_ work on PowerShell on Windows, because that adds an _extra_ level of escaping that is often hard to deal with, but that's not a new problem nor something we can easily address in the documentation here.

With that said, the lead-in text here suggests that all of these forms are things the user might expect to see in the output of other commands, and so the escaping here is making that no longer strictly true. Perhaps we should hint that with some parenthetical text:

> * `aws_instance.baz[\"key\"]` (quotes in resource addresses must be escaped on the command line so that they are not interpreted by your shell)